appengine-standard-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -48,20 +36,18 @@ limitations under the License.
         <include>**/*.launch</include>
       </includes>
     </fileSet>
-    <fileSet encoding="UTF-8">
+    <fileSet filtered="true" encoding="UTF-8">
       <directory></directory>
       <includes>
         <include>nbactions.xml</include>
         <include>README.md</include>
+        <include>build.gradle</include>
```

Comment:
We'll find out.  There currently isn't a way to do gradle archetypes.  (I'm included to just do a shell script.  But I find Gradle faster than maven.  So, I'd upgrade.